pandas/io/formats/style_render.py

```diff
@@ -365,7 +372,12 @@ def _translate_header(self, sparsify_cols: bool, max_cols: int):
                 )
                 head.append(header_row)
 
-        # 2) index names
+        # 2) Descriptor calcs
+        for r, descriptor in enumerate(self.descriptors):
+            descriptor_row = self._generate_descriptor_row((r, descriptor), max_cols)
```

Comment:
Why pass a tuple to this function?  It would be better to pass the row and the descriptor as separate arguments since there is no gain to repacking them, since you immediately unpack them in the function.

Reply:
agreed, but all the other generator methods use this format (rightly or wrongly) so I stuck to the same format for consistency. is minor either way.